mongodb-advanced/components/people-component.js

```diff
@@ -0,0 +1,28 @@
+/*!
+ * Copyright (c) 2016-2017 Digital Bazaar, Inc. All rights reserved.
+ */
+export default {
+  controller: Ctrl,
+  templateUrl: 'mongodb-advanced/people-component.html'
+};
+
+function Ctrl($scope, exPeopleService, brAlertService) {
+  const self = this;
+  self.showMyModal = false;
+
+  self.getPeople = () => exPeopleService.getPeople().then(people =>
+    self.savedPeople = people)
+    .catch(err => brAlertService.add('error', err));
+
+  self.getPeople();
```

Comment:
This should be:
```
self.$onInit(() => self.getPeople());
```

Reply:
Replacing...```self.getPeople()```
with ```self.onInit(() => self.getPeople());```
results in a browser console error ```TypeError: self.onInit is not a function```

Reply:
@jeiholzer sorry, wrote out the wrong syntax.  Please try to make use of web resources such as this when possible: https://toddmotto.com/angular-1-5-lifecycle-hooks#using-oninit

Reply:
also, I don't know if you've omitted the `$` in the code you put in your comment, but it IS `self.$onInit` with the $.  And of course because we do `self = this;` at the top of each controller, we use `self.foo` instead of `this.foo` as shown in the toddmotto.com example.